src/internal/operations/maintenance_test.go

```diff
@@ -65,3 +91,155 @@ func (suite *MaintenanceOpIntegrationSuite) TestRepoMaintenance() {
 	err = mo.Run(ctx)
 	assert.NoError(t, err, clues.ToCore(err))
 }
+
+func TestMaintenanceOpNightlyIntegrationSuite(t *testing.T) {
```

Comment:
Ever so slightly less wordy this way.
```suggestion
func TestMaintenanceOpNightlySuite(t *testing.T) {
```